src/nvim/ops.c

```diff
@@ -2524,23 +2517,24 @@ static void yank_copy_line(yankreg_T *reg, struct block_def *bd, long y_idx)
 /*
  * Put contents of register "regname" into the text.
  * Caller must check "regname" to be valid!
- * "flags": PUT_FIXINDENT	make indent look nice
- *	    PUT_CURSEND		leave cursor after end of new text
- *	    PUT_LINE		force linewise put (":put")
+ * "flags": PUT_FIXINDENT     make indent look nice
+ *          PUT_CURSEND       leave cursor after end of new text
+ *          PUT_LINE          force linewise put (":put")
     dir: BACKWARD for 'P', FORWARD for 'p' */
 void do_put(int regname, yankreg_T *reg, int dir, long count, int flags)
 {
-  char_u      *ptr;
-  char_u      *newp, *oldp;
+  char_u *ptr;
+  char_u *newp;
+  char_u *oldp;
   int yanklen;
-  int totlen = 0;                       /* init for gcc */
```

Comment:
I would like this to be `size_t` but I don't understand under what circumstance `count` can be negative. I _think_ only in the special case `regname == '.'`, at least the rest of the code it looks like `count = -1` has the same effect as `0`.
(even in the `.` case I don't understand, as `".` is _always_ charwise so why would you `o`/`O` it?)